### PR TITLE
fix select_pattern on SLE12

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

**Comment by mimi1vx** [comment]
mr#76 on gitlab has needed needle

File: tests/installation/select_patterns.pm [hunk 1]
```diff
@@ -26,6 +26,9 @@ sub accept3rdparty {
 sub movedownelseend {
     my $ret = wait_screen_change {
         send_key 'down';
+        if (check_screen('pattern-too-low', 5)) {
+            assert_and_click('pattern-too-low', 'left', 30, 0.5);
+        }
```
Comment (coolo) [inline tests/installation/select_patterns.pm]: I would like an explanation in a comment why you pass all these parameters

Comment (dzedro) [inline tests/installation/select_patterns.pm]: yes some comment would help, click because of incomplete last line or something :)

Comment (coolo) [inline tests/installation/select_patterns.pm]: that part is clear to me, but why the long times is not

Comment (mimi1vx) [inline tests/installation/select_patterns.pm]: this is seen firstime with qam-allpatterns+we test on SLE12

Comment (sysrich) [inline tests/installation/select_patterns.pm]: But why the ", 30, 0.5" on the assert_and_click?

Comment (coolo) [inline tests/installation/select_patterns.pm]: beside that - having a check_screen within a wait_screen_change sounds like a bad idea

File: tests/installation/select_patterns.pm [hunk 2]
```diff
@@ -31,6 +31,14 @@ sub movedownelseend {
     last if (!$ret);
 }
 
+sub check12qtbug {
+    if (check_var('VERSION', '12')) {
```
Comment (coolo) [inline tests/installation/select_patterns.pm]: good thing we passed openSUSE 12 long ago ;)

**Pull request merged**